### DISCUSSION TITLE
Build the latest iperf 3 in raspberry pi 4?

### DISCUSSION THREAD

**Discussion by acdcjavier** [esnet/iperf#1357] [General]
Hi!
A short question?
Exist any way or posibility to compile and install the latest iperf3 (3.11) to raspberry pi 4?

Currently, the latest official version available found in rp4 is 3.6 but I need to update it to a newer version because I need to use this command this flag:
`--timestamp` 
option which is not implemented in version 3.6 yet.

**Comment by davidBar-On** [comment]
You may be able to build iperf3 yourself.  See for example [these instructions for how to do that](https://forums.raspberrypi.com/viewtopic.php?t=313629#p1876084). iperf3 source tar files for the different releases are available from [here](http://software.es.net/iperf/news.html).

**Comment by rahmanuh** [comment]
Hi,
I built it myself for Raspberry Pi 4. Following the build-guide works out of the box. Just download the 3.11 package, then follow this https://github.com/esnet/iperf#building.

**Comment by userdocs** [comment]
I make fully static builds for all Alpine supported arches here, These would work PI devices and more as they work on any Linux based OS with matching arch.

https://github.com/userdocs/iperf3-static/releases/latest

| Alpine Arch | Docker platform arch |  Docker hub image   |
| :---------: | :------------------: | :-----------------: |
|    armhf    |     linux/arm/v6     | arm32v6/alpine:edge |
|    armv7    |     linux/arm/v7     | arm32v7/alpine:edge |
|   aarch64   |     linux/arm64      | arm64v8/alpine:edge |
|   ppc64le   |    linux/ppc64le     | ppc64le/alpine:edge |
|    s390x    |     linux/s390x      |  s390x/alpine:edge  |
|     x86     |      linux/i386      |  i386/alpine:edge   |
|   x86_64    |     linux/amd64      |  amd64/alpine:edge  |
